libweston/backend-rdp/rdprail.c

```diff
@@ -59,163 +59,174 @@ static void rdp_rail_destroy_window(struct wl_listener *listener, void *data);
 static void rdp_rail_schedule_update_window(struct wl_listener *listener, void *data);
 static void rdp_rail_dump_window_label(struct weston_surface *surface, char *label, uint32_t label_size);
 
+struct lang_GUID {
+	uint32_t Data1;
+	uint16_t Data2;
+	uint16_t Data3;
+	BYTE Data4_0;
+	BYTE Data4_1;
+	BYTE Data4_2;
+	BYTE Data4_3;
+	BYTE Data4_4;
+	BYTE Data4_5;
+	BYTE Data4_6;
+	BYTE Data4_7;
+};
+
 struct rdp_rail_dispatch_data {
 	struct rdp_loop_task task_base;
 	freerdp_peer *client;
 	union {
-		RAIL_SYSPARAM_ORDER u_sysParam;
-		RAIL_SYSCOMMAND_ORDER u_sysCommand;
-		RAIL_ACTIVATE_ORDER u_activate;
-		RAIL_EXEC_ORDER u_exec;
-		RAIL_WINDOW_MOVE_ORDER u_windowMove;
-		RAIL_SNAP_ARRANGE u_snapArrange;
-		RAIL_GET_APPID_REQ_ORDER u_getAppidReq;
-		RAIL_LANGUAGEIME_INFO_ORDER u_languageImeInfo;
+		RAIL_SYSPARAM_ORDER sys_param;
+		RAIL_SYSCOMMAND_ORDER sys_command;
+		RAIL_ACTIVATE_ORDER activate;
+		RAIL_EXEC_ORDER exec;
+		RAIL_WINDOW_MOVE_ORDER window_move;
+		RAIL_SNAP_ARRANGE snap_arrange;
+		RAIL_GET_APPID_REQ_ORDER get_appid_req;
+		RAIL_LANGUAGEIME_INFO_ORDER language_ime_info;
 #ifdef HAVE_FREERDP_RDPAPPLIST_H
-		RDPAPPLIST_CLIENT_CAPS_PDU u_appListCaps;
-#endif // HAVE_FREERDP_RDPAPPLIST_H
+		RDPAPPLIST_CLIENT_CAPS_PDU app_list_caps;
+#endif /* HAVE_FREERDP_RDPAPPLIST_H */
 	};
 };
 
 #define RDP_DISPATCH_TO_DISPLAY_LOOP(context, arg_type, arg, callback) \
 	{ \
-		freerdp_peer *client = (freerdp_peer*)(context)->custom; \
-		RdpPeerContext *peerCtx = (RdpPeerContext *)client->context; \
-		struct rdp_backend *b = peerCtx->rdpBackend; \
+		freerdp_peer *client = (context)->custom; \
+		RdpPeerContext *peer_ctx = (RdpPeerContext *)client->context; \

```

Comment:
Just would like to double check, the use of "peerCtx" is from original weston, and still is at upstream https://gitlab.freedesktop.org/wayland/weston/-/blob/main/libweston/backend-rdp/rdp.c#L1506.

Reply:
Yep, rdp.c has been doing this, and shouldn't be (the rest of the project does_variables_with_underscores instead). Making the existing code be consistent with the rest of weston would be nice, but it would also impose a burden on anyone carrying out of tree patches. We'd like newly landed code to be more conformant though.